draco/analysis/beam.py

```diff
@@ -119,30 +125,24 @@ def process(self, data, beam):
         # correct for the rotation in this way, since we have already collapsed
         # over NS baselines. However, this partial correction should be pretty good
         # for small rotation angles and for sources near meridian.
-        rot = np.radians(self.telescope.rotation_angle)
-        v = np.sin(rot) * u
-        u = np.cos(rot) * u
-
-        # Calculate the phase
-        phi = interferometry.fringestop_phase(
-            arr_ha, np.radians(self.telescope.latitude), arr_dec, u, v
-        ).conj()
+        v = np.sin(self.rotation_angle) * u
+        u = np.cos(self.rotation_angle) * u
 
         # Reshape the beam datasets to match the output container.
         # The output weight dataset does not have an el axis, use the
         # average non-zero value of the weight along the el direction.
-        bweight = beam.weight[:]
+        bweight = beam.weight[:].local_array
         bweight = np.sum(bweight, axis=-2) * tools.invert_no_zero(
             np.sum(bweight > 0, axis=-2, dtype=np.float32)
         )
 
         # Transpose the first two dimensions from (freq, pol) to (pol, freq)
         bweight = bweight.swapaxes(0, 1)
-        bvis = beam.beam[:].swapaxes(0, 1)
+        bvis = beam.beam[:].local_array.swapaxes(0, 1)
 
         # Create output container
         out = containers.HybridVisStream(
-            ra=nra,
+            ra=ra,
```

Comment:
In the celestial beam case, `ra` is defined based on the `phi` axis of the beam container, which doesn't span the full RA range and causes a crash later on. I will fix this and set the test job running again.